apps/settings/lib/SetupChecks/PhpOpcacheSetup.php

```diff
@@ -114,6 +114,11 @@ protected function getOpcacheSetupRecommendations(): array {
 	}
 
 	public function run(): SetupResult {
+		// Skip OPcache checks if running from CLI
+		if (PHP_SAPI === 'cli') {
+			return SetupResult::success($this->l10n->t('Checking from CLI, OPcache checks have been skipped.'));
```

Comment:
Just in case is might be ever wanted for debugging, development or CI, an alternative would be to do run those checks, if OPcache is enabled for CLI:
```suggestion
if (PHP_SAPI === 'cli' && !$this->iniGetWrapper->getBool('opcache.enable_cli')) {
	return SetupResult::success($this->l10n->t('OPcache checks are skipped since it is disabled for CLI.'));
```
It could be also done in the last `else` block of `getOpcacheSetupRecommendations()`, to at last do those checks which are still possible from CLI: if the module is enabled, and shared memory based cache enabled for web, theoretically also saved comments etc. However, since PHP modules can be enabled/disabled per SAPI, this bears the risk of false positives.